var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -679,30 +688,24 @@ def build(self, spec, prefix):
 
         build_pip_package = Executable(
             'bazel-bin/tensorflow/tools/pip_package/build_pip_package')
-        build_pip_package(tmp_path)
+        buildpath = join_path(self.stage.source_path, 'spack-build')
+        build_pip_package('--src', buildpath)
 
     def install(self, spec, prefix):
-        with working_dir('spack-build', create=True):
-            for fn in glob.iglob(join_path(
-                    '../bazel-bin/tensorflow/tools/pip_package',
-                    'build_pip_package.runfiles/org_tensorflow/*')):
-                dst = os.path.basename(fn)
-                if not os.path.exists(dst):
-                    os.symlink(fn, dst)
-            for fn in glob.iglob('../tensorflow/tools/pip_package/*'):
-                dst = os.path.basename(fn)
-                if not os.path.exists(dst):
-                    os.symlink(fn, dst)
-
-            # macOS is case-insensitive, and BUILD file in directory
-            # containing setup.py causes the following error message:
-            #     error: could not create 'build': File exists
-            # Delete BUILD file to prevent this.
-            os.remove('BUILD')
+        buildpath = join_path(self.stage.source_path, 'spack-build')
+        with working_dir(buildpath):
 
             setup_py('install', '--prefix={0}'.format(prefix),
                      '--single-version-externally-managed', '--root=/')
 
+            site_packages_dir = join_path(
+                prefix.lib,
+                ('python' + str(self.spec['python'].version.up_to(2))),
+                'site-packages')
+            fn = glob(join_path(site_packages_dir, "tensorflow-*"))
+            incpath = join_path(fn[0], "tensorflow/include")
```

Comment:
For me, this installs all header files to:
```
lib/python3.7/site-packages/tensorflow-2.1.0-py3.7.egg-info/tensorflow/include/tensorflow
```
Is this what you intended?

Reply:
yes - this was what seemed to be required via horovod's use of things like this:
```
    import tensorflow as tf
    tf_inc = tf.sysconfig.get_include()

```

Reply:
going to double check this anyways. give me a bit.

Reply:
Too late, you can always submit a follow-up PR to fix it if it's incorrect. It's not like you broke anything since it never worked before.

Reply:
ah lol